tools/respecDocWriter.js

```diff
@@ -133,9 +132,9 @@ async function checkReSpecVersion(page) {
   const [mayor] = version;
   // The exportDocument() method only appeared in vesion 18.
   if (mayor < 18) {
-    let msg =
-      "👴🏽  Ye Olde ReSpec version detected! " +
-      colors.debug("Sorry, we only support ReSpec version 18.0.0 onwards.\n");
+    let msg = `👴🏽  Ye Olde ReSpec version detected! ${colors.debug(
```

Comment:
Coincidentally, we should remove this in another PR.

Reply:
Or should we set it as 20.10.0, as another line checks it to fallback to older module?

Reply:
Nah, it's been over a year. I think that's an acceptable length of time/warning for anyone on.